CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
```

Comment:
I don't think we need this file. The one to add when _using_ the template is good, but release notes for this particular project can be kept entirely on… well… github release notes. I think.

Reply:
done